src/mongoose_graphql.erl

```diff
@@ -88,8 +88,14 @@ execute(Ep, OpName, Doc)  ->
 
 % Internal
 
+schema_global_patterns(SchemaDir) ->
+    [schema_pattern(SchemaDir), schema_pattern(global)].
+
 schema_pattern(DirName) ->
```

Comment:
Could you spec this? Also, currently it accepts the atom `global` or a string. It's a bit inconsistent.